apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/components/template-selector-preview.js

```diff
@@ -103,40 +69,29 @@ const TemplateSelectorPreview = ( { blocks, viewportWidth, title } ) => {
 		return () => {
 			window.removeEventListener( 'resize', refreshPreview );
 		};
-	}, [ blocks, updatePreviewViewport ] );
-
-	if ( isEmpty( blocks ) || ! isArray( blocks ) ) {
-		return (
-			<div className={ classnames( 'template-selector-preview', 'is-blank-preview' ) }>
-				<div className="template-selector-preview__placeholder">
-					{ __( 'Select a layout to preview.', 'full-site-editing' ) }
-				</div>
-			</div>
-		);
-	}
+	}, [ blocks ] );
 
 	return (
 		/* eslint-disable wpcalypso/jsx-classname-namespace */
-		<div className="template-selector-preview">
-			<Disabled>
-				<div ref={ ref } className="edit-post-visual-editor">
-					<div className="editor-styles-wrapper">
-						<div className="editor-writing-flow">
-							<PreviewTemplateTitle title={ title } scale={ titleTransform.scale } />
-							<div
-								className="template-selector-preview__offset-correction"
-								style={ { top: titleTransform.offset } }
-							>
-								<BlockPreview
-									key={ recompute }
-									blocks={ blocks }
-									viewportWidth={ previewViewport }
-								/>
-							</div>
-						</div>
-					</div>
-				</div>
-			</Disabled>
+		<div className={ `template-selector-preview ${ ! blocks.length ? 'not-selected' : '' }` }>
+			<BlockTemplatePreview
+				key={ recompute }
+				blocks={
+					! blocks.length
+						? createBlock( 'core/paragraph', {
+							content: __( 'Select a layout to preview.', 'full-site-editing' ),
+						} )
+						: [
+								createBlock( 'core/heading', {
```

Comment:
Note that [`core/site-title`](https://github.com/WordPress/gutenberg/tree/a36411a5655d8948e4abbaa4bcf487938facbeb4/packages/block-library/src/site-title) is now available too. :-)

Reply:
yes, I think we can create a block in just put it in the blocks list.

Reply:
is it available?

Reply:
I mean I'm not able to create a `core/site-title` block. 

```
> wp.data
  .select( 'core/blocks' )
  .getBlockTypes()
  .map( block => block.name )
  .filter( name => name.match( /title/ ) );
```

it returns an empty `[]` array

Reply:
Ah, you're right. We have Gutenberg 7.3.0 on wpcom currently (about to be updated to 7.6.0, p7DVsv-8c5-p2) and it likely doesn't have it yet.

Even once that's rolled out, I'd expect to have Atomic sites which are on an older versions of Gutenberg or have disabled it completely. I don't know if the whole FSE plugin with SPTs loads in that case tho.

Reply:
@simison Can you spin out a separate Issue for this one so it can be tracked? I think it's unlikely that we'll tackle it here. Much appreciated.